kong/plugins/cors/handler.lua

```diff
@@ -181,7 +181,7 @@ local function configure_credentials(conf, allow_all, header_filter)
 
   -- Access-Control-Allow-Origin is '*', must change it because ACAC cannot
   -- be 'true' if ACAO is '*'.
-  local req_origin = kong.request.get_header("origin")
+  local req_origin = headers["origin"]
```

Comment:
Could you confirm it is ok?

Reply:
Keep using `kong.request.get_header("origin")`